### DISCUSSION TITLE
Sort by End of Life (EOL)

### DISCUSSION THREAD

**Discussion by ReidCoCam** [snipe/snipe-it#12413] [Ideas]
Hello hello.

Would it be possible to add an ascending/descending option for the End of Life (EOL) tab in the Asset view? For some reason, every other column that I can see has sorting options, but EOL doesn't. This would make it significantly easier for me and my team to track when devices are due for a refresh. I've attached an image of my Asset view to show what I mean. 

Or having an automated monthly report of all devices coming up on their EOL period, but I think the sorting option would be best.

Thank you.
<img width="2476" alt="Snipe_EOL" src="https://user-images.githubusercontent.com/123576672/214617734-49fe4529-3833-4a5a-af55-a6e5902f0cc1.png">

**Comment by najirod** [comment]
There is PR for sorting - https://github.com/snipe/snipe-it/pull/12393
and for report - https://github.com/snipe/snipe-it/pull/11296
